### PR TITLE
Added option to suppress comments

### PR DESCRIPTION
**Pull request by sfc-gh-jsikorski**
### Pre-review checklist
   * [x] I've confirmed that instructions included in README.md are still correct after my changes in the codebase.
   * [ ] I've added or updated automated unit tests to verify correctness of my new code.
   * [x] I've added or updated integration tests to verify correctness of my new code.
   * [x] I've confirmed that my changes are working by executing CLI's commands manually.
   * [x] I've confirmed that my changes are up-to-date with the target branch.
   * [x] I've described my changes in the release notes.
   * [x] I've described my changes in the section below.

### Changes description
Previously, we didn't use the `remove_comments` option of snow connector's methods for sql execution. So comments were always shown. I've changed this behaviour, so comments are hidden by default.
Also, added `-s` flag to `snow sql` command, to show comments.

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -50,6 +50,7 @@
 * Added support for runtime version in snowpark procedures ad functions.
 * You can include previously uploaded packages in your functions, by listing them under `imports` in `snowflake.yml`
 * Added more options to `snow connection add` - now you can also specify authenticator and path to private key
+* Added `-s` flag to `snow sql` command, that shows comments. Comments, now are not shown by default
```
Comment (sfc-gh-astus) [inline RELEASE-NOTES.md:53]: Maybe `Added "--show-comments"`?

File: src/snowcli/plugins/sql/manager.py [hunk 2]
```diff
@@ -11,7 +11,11 @@
 
 class SqlManager(SqlExecutionMixin):
     def execute(
-        self, query: Optional[str], file: Optional[Path], std_in: bool
+        self,
+        query: Optional[str],
+        file: Optional[Path],
+        std_in: bool,
+        remove_comments: bool = False,
```
Comment (sfc-gh-astus) [inline src/snowcli/plugins/sql/manager.py:18]: `Comments, now are not shown by default` 

Remove comments shouldn't be default True?

File: src/snowcli/plugins/sql/commands.py [hunk 3]
```diff
@@ -36,6 +36,9 @@ def execute_sql(
         "-i",
         help="Read the query from standard input. Use it when piping input to this command.",
     ),
+    show_comments: Optional[bool] = typer.Option(
+        False, "--show-comments", "-s", help="Removes comments from the input query"
```
Comment (sfc-gh-astus) [inline src/snowcli/plugins/sql/commands.py:40]: Do we need `-s` alias? It can cause conflicts

File: RELEASE-NOTES.md (hunk 1 above)
Comment (sfc-gh-turbaszek) [reply RELEASE-NOTES.md:53]: How about `--comments` and `--no-comments`? I think typer will support this out of the box:
https://typer.tiangolo.com/tutorial/parameter-types/bool/

Comment (sfc-gh-jsikorski) [reply RELEASE-NOTES.md]: Done

File: src/snowcli/plugins/sql/manager.py (hunk 2 above)
Comment (sfc-gh-jsikorski) [reply src/snowcli/plugins/sql/manager.py]: Done

File: src/snowcli/plugins/sql/commands.py (hunk 3 above)
Comment (sfc-gh-jsikorski) [reply src/snowcli/plugins/sql/commands.py]: Removed

File: src/snowcli/plugins/sql/manager.py [hunk 4]
```diff
@@ -29,4 +33,6 @@ def execute(
             query = file.read_text()
 
         single_statement = len(list(split_statements(StringIO(query)))) == 1
-        return single_statement, self._execute_string(query)
+        return single_statement, self._execute_string(
+            query, remove_comments=remove_comments
```
Comment (sfc-gh-astus) [inline src/snowcli/plugins/sql/manager.py:37]: Maybe keep our name `show_comments` until the very end and do here

```
remove_comments=not show_comments
```

WDYT? We will be consistency on our side

File: src/snowcli/plugins/sql/manager.py [hunk 5]
```diff
@@ -11,7 +11,11 @@
 
 class SqlManager(SqlExecutionMixin):
     def execute(
-        self, query: Optional[str], file: Optional[Path], std_in: bool
+        self,
+        query: Optional[str],
+        file: Optional[Path],
+        std_in: bool,
+        show_comments: bool = True,
```
Comment (sfc-gh-astus) [inline src/snowcli/plugins/sql/manager.py:18]: `Comments, now are not shown by default` 😄

**Review by sfc-gh-astus — DISMISSED** [review]

**Pull request closed without merging**